Sum-of-two-lowest-positive-integers/sumTwoSmallestNumbers.js

```diff
@@ -0,0 +1,8 @@
+const sumTwoSmallestNumbers = (numbers) => {
+  numbers.sort(function (a, b) {
```

Comment:
please refactor this `function` to ES6 anonymous function (`(a,b) => ...`) and return value in one line instead of defining a body of a function